recipe/patches/0003-Remove-all-dependencies-from-setup.py.patch

```diff
@@ -77,19 +89,32 @@ index 1b8c8e156..c6ee62a96 100644
 -        "scikit-image",
 -        "pyyaml",
 -        "scipy",
+-        "typer",
+-        "rich",
```

Comment:
They show up here, which was the intent of this patch (I'd like the tool you propose as well, but as long as we don't have it...)